test/parsing-test.js

```diff
@@ -30,6 +30,18 @@ describe('Messages parsing', function () {
         sendGst.restore()
         getRoom.restore()
     })
+    it('return +todo gdocs url', function () {
+      return room.user.say('mikanebu', "bot which todos gdocs").then(function () {
+        assert.equal(room.messages[1][1].substr(0, 14), "@mikanebu http")
+        assert.equal((room.messages).length, 2)
+      })
+    })
+    it('return +todo gist url', function () {
+      return room.user.say('mikanebu', "bot which todos gist").then(function () {
+        assert.equal(room.messages[1][1].substr(0, 14), "@mikanebu http")
```

Comment:
same here
plus this test does not me sure that it prints link for gist and not for gdoc (it tests against same message that for gist)